tests/unit/ops/cuda/test_cuda_backward.py

```diff
@@ -8,12 +8,16 @@
 from deepspeed import DeepSpeedTransformerLayer, DeepSpeedTransformerConfig
 from unit.modeling import BertConfig, BertLayerNorm, BertEncoder as BertEncoderPostln
 from unit.modelingpreln import BertEncoder as BertEncoderPreln
+from unit.common import DistributedTest
 
 #if not deepspeed.ops.__installed_ops__['transformer']:
 #pytest.skip(
 #    "transformer kernels are temporarily disabled because of unexplained failures",
 #    allow_module_level=True)
 
+if not torch.cuda.is_available():
+    pytest.skip("Only supported on CUDA environments", allow_module_level=True)
```

Comment:
Is this necessary? It should be skipped in `DistributedTest` class with these lines:
```python
if self.requires_cuda_env and not torch.cuda.is_available():
    pytest.skip("only supported in CUDA environments.")
```

Reply:
Not necessary, I forgot to remove.